Makefile

```
@@ -13,6 +13,14 @@ build:
 	GOFLAGS=$(GOFLAGS) go build $(BUILD_FLAGS) -o ./bin/status-term-client .
 .PHONY: build
 
+# XXX: Multiple ldflags a bit brittle, keeping it simple by having separate build target for now.
+# See https://github.com/golang/go/issues/29053
+# TODO: parameterize BUILD_FLAGS properly with a Nimbus build flag
+build-nimbus: GOFLAGS ?= "-mod=vendor"
+build-nimbus:
+	GOFLAGS=$(GOFLAGS) go build -ldflags="-r /usr/local/lib" -o ./bin/status-term-client .
```

Maybe we can have a build tag:
```suggestion
	GOFLAGS=$(GOFLAGS) go build -tags nimbus -ldflags="-r /usr/local/lib" -o ./bin/status-term-client .
```

---

`/usr/local/lib` probably should be configurable.

---

Agee in general, but this is a spike, not needed for now. Proper installations can come in a separate issue

---

How do build tags work in Go? Cursory googling doesn't show a lot of detail. Here's how it works in Nim: https://github.com/oskarth/nim-secp256k1/commit/af658db8c3062d907f2b927f3229735639220259